app/javascript/features/form/components/form.tsx

```diff
@@ -339,12 +344,11 @@ const Form = (props: Props) => {
                   予約
                 </Button>
               </Box>
-              <Backdrop
-                className={classes.backdrop}
-                open={isSubmitting || isLoading}
-              >
-                <CircularProgress color="inherit" />
-              </Backdrop>
+              <LoadingForm
+                menus={menus}
+                isLoading={isLoading}
+                title={title}
+              ></LoadingForm>
```

Comment:
```suggestion
              <LoadingForm
                menus={menus}
                isLoading={isLoading}
                title={title}
              />
```
子Componentがない場合は↑みたいに書けます